.build/build.ps1

```diff
@@ -39,6 +64,32 @@ task Publish.Module Build.Module, {
     {
         throw 'Missing required parameter "Version".'
     }
+
+    if (-not $ApiKey)
+    {
+        throw 'Missing required parameter "ApiKey".'
+    }
+
+    try
+    {
+        Import-Module -Name $modulePath -ErrorAction 'Stop'
+        $moduleInfo = Get-Module -Name $moduleName -ErrorAction 'Stop'
+    }
+    catch
+    {
+        throw $_
+    }
+
+    $parameters = @{}
+    foreach ($key in @('Tags', 'IconUri', 'ProjectUri', 'LicenseUri', 'ReleaseNotes'))
+    {
+        if ($moduleInfo.$key)
+        {
+            $parameters.Add($key, $moduleInfo.$key)
+        }
+    }
+
+    Publish-Module -Name $moduleName -Repository 'PSGallery' -NuGetApiKey $ApiKey -Force -AllowPrerelease @parameters
```

Comment:
Are you publishing as part of the build? Shouldn't we do that separately so we can run local and PR builds?

Reply:
Nothing is currently pointing to this task. That's why I was asking about how we want to handle the publishing, whether it be manual or automated. Should probably discuss offline

Reply:
I was assuming we'd run the build with tests as part of a PR to at least validate what we can. It's okay if that's separate.